commcare_connect/opportunity/views.py

```
@@ -503,40 +503,29 @@ def review_visit_export(request, org_slug, opp_id):
 def export_status(request, org_slug, task_id):
-    task = AsyncResult(task_id)
-    task_meta = task._get_task_meta()
-    opportunity_id = task_meta.get("args")[0]
-    # Make sure opportunity exists for the given org_slug
-    get_opportunity_or_404(org_slug=org_slug, pk=opportunity_id)
-    status = task_meta.get("status")
-    progress = {"complete": status == CELERY_TASK_SUCCESS, "message": get_task_progress_message(task)}
-    if status == "FAILURE":
-        progress["error"] = task_meta.get("result")
-    return render(
+    def ownership_check(request, task_meta):
```

Ruff (0.14.13)

Unused function argument: `request`

(ARG001)